requirements/test-requirements.txt

```diff
@@ -4,564 +4,193 @@
-    #   dropbox
-    #   pygithub
-    #   requests-mock
-    #   requests-oauthlib
-    #   stripe
-    #   tinys3
-    #   twilio
-rjsmin==1.0.12
-    # via django-compressor
-s3transfer==0.3.3
-    # via boto3
-schema==0.7.2
-    # via -r requirements.in
-sentry-sdk==0.19.4
-    # via -r requirements.in
-sh==1.14.1
-    # via -r requirements.in
-simpleeval==0.9.10
-    # via -r requirements.in
-simplejson==3.17.2
-    # via
-    #   -r requirements.in
-    #   django-prbac
-six==1.15.0
-    # via
-    #   -r requirements.in
-    #   cryptography
-    #   django-braces
-    #   django-simple-captcha
-    #   django-statici18n
-    #   django-transfer
-    #   django-websocket-redis
-    #   dropbox
-    #   ethiopian-date-converter
-    #   eulxml
-    #   freezegun
-    #   ghdiff
-    #   html5lib
-    #   isodate
-    #   jsonfield
-    #   jsonobject
-    #   jsonobject-couchdbkit
-    #   jsonpath-rw
-    #   mock
-    #   protobuf
-    #   pyopenssl
-    #   python-dateutil
-    #   pyzxcvbn
-    #   qrcode
-    #   quickcache
-    #   requests-mock
-    #   sqlalchemy-postgres-copy
-    #   tenacity
-    #   twilio
-    #   unittest2
-smartypants==2.0.1
-    # via pycco
-socketpool==0.5.3
-    # via -r requirements.in
-sortedcontainers==2.3.0
-    # via intervaltree
-soupsieve==2.0.1
-    # via beautifulsoup4
-sqlagg==0.17.2
-    # via -r requirements.in
-sqlalchemy-postgres-copy==0.5.0
-    # via -r test-requirements.in
-sqlalchemy==1.3.19
-    # via
-    #   -r requirements.in
-    #   alembic
-    #   sqlagg
-    #   sqlalchemy-postgres-copy
-sqlparse==0.3.1
-    # via django
-stripe==2.54.0
-    # via -r requirements.in
-suds-jurko==0.6
-    # via -r requirements.in
-tenacity==6.2.0
-    # via ddtrace
-testil==1.1
-    # via -r test-requirements.in
-text-unidecode==1.3
-    # via
-    #   -r requirements.in
-    #   faker
-tinycss2==1.0.2
-    # via
-    #   cairosvg
-    #   cssselect2
-    #   weasyprint
-tinys3==0.1.12
-    # via -r requirements.in
-traceback2==1.4.0
-    # via unittest2
-tropo-webapi-python==0.1.3
-    # via -r requirements.in
-turn-python==0.0.1
-    # via -r requirements.in
-twilio==6.5.1
-    # via -r requirements.in
-ua-parser==0.10.0
-    # via user-agents
-unittest2==1.1.0
-    # via -r test-requirements.in
-urllib3==1.25.10
-    # via
-    #   botocore
-    #   elasticsearch
-    #   elasticsearch2
-    #   elasticsearch7
-    #   py-kissmetrics
-    #   requests
-    #   sentry-sdk
-user-agents==2.2.0
-    # via django-user-agents
-vine==1.3.0
-    # via amqp
-weasyprint==51
-    # via -r requirements.in
-webencodings==0.5.1
-    # via
-    #   cssselect2
-    #   html5lib
-    #   tinycss2
-werkzeug==1.0.1
-    # via -r requirements.in
-wrapt==1.12.1
-    # via deprecated
-xlrd==1.0.0
-    # via -r requirements.in
-xlwt==1.3.0
-    # via -r requirements.in
-xmlsec==1.3.9
-    # via python3-saml
+alembic==1.4.3            # via -r requirements.in
+amqp==2.6.1               # via kombu
```

Comment:
I think you need to upgrade pip-tools in your local environment and then run `make requirements` again to fix the format of these files.

Reply:
thanks yea i hadn't done it. looks like there were other tests breaking too >_<